src/devices/Mcp960x/Mcp960x.csproj

```diff
@@ -12,7 +12,6 @@
   <ItemGroup>
     <Compile Include="*.cs" />
     <None Include="README.md" />
```

Comment:
I think this shouldn't be part of this PR - presumably you did some kind of pull + rebase on your branch if this wasn't intentional

Reply:
Fixed this issue

Reply:
seems still shows up on the PR, also below

Reply:
It is fine to keep, in fact I believe not doing so would break the build so I don't mind if this fix is also checked in.